department-of-veterans-affairs/caseflow
# APPEALS-18739

Resolves [APEALS-18739](https://vajira.max.gov/browse/APPEALS-18739)

# Description
Adds a link to case search in the Switch Product dropdown

## Acceptance Criteria
- [ ] When user selects the Caseflow Search cases menu option from the drop-down menu navigation, they will be directed to the Veteran Case Search (same address as currently existing link available in Caseflow Queue and Caseflow Intake - 'Search Cases').
- [ ] If the user selects ‘Caseflow Search cases’ menu option:
    - [ ] While currently on a Case Details or Edit Claim Issues page, they will be returned to ‘Veteran Case Search’ page.  
    - [ ] While on the main Veteran Case Search landing page, the page will refresh the ‘Veteran Case Search’ page.
- [ ] If a user selects a menu navigation option, there will be no modals displayed if a user has entered data to the screen unless they select a ‘Cancel’ link related to their current process.


## Testing Plan

1. Log in as CAMOUSER
2. Navigate to the queue
3. In the top left 'Switch product' navigation menu, 'Caseflow Search cases' should be there
4. Clicking it should navigate to the case search page


# Frontend
## User Facing Changes
<img width="1257" alt="image" src="https://github.com/department-of-veterans-affairs/caseflow/assets/17840614/f9c9ef93-db2a-47f9-b253-57a2096c6e37">

## Tests
### Test Coverage
Did you include any test coverage for your code? Check below:

### Code Climate
Your code does not add any new code climate offenses? If so why?
